app/services/hawkular_proxy_service.rb

```diff
@@ -8,10 +8,16 @@ def initialize(provider_id, controller)
     @params = controller.params
     @ems = ManageIQ::Providers::ContainerManager.find(@provider_id) unless @provider_id.blank?
     @tenant = @params['tenant'] || '_system'
+
+    @cli = ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClient.new(@ems, @tenant)
   end
 
   def client
```

Comment:
@yaacov is this still necessary? If it's used only by `metric_definitions` I think it can be dropped.